### DISCUSSION TITLE
Quickdraw achievement badge doesn't show up on profile

### DISCUSSION THREAD

**Discussion by abir75mal** [community/community#64882] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I've tried different issue closing and PR merging scenarios, private and public, but I still don't see it.
I also have "Include private contributions on my profile" set.
Any thoughts?
Thanks!

**Comment by artiomtz** [comment]
1. You get it if you close a PR/issue within 5 minutes. The PR has to be successfully merged.
2. Make sure you have "Contributions & Activity" set to "Include private contributions on my profile" in the Settings > Public profile.
3. Click the Achievements title on your profile page. If it's already in Achievements but not shown, open the achievement and toggle the eye icon for it to be shown on the profile page.

**Reply by abir75mal** [reply]
Oh it was number 3. Thanks!

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
